.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
 env:
   - SYMFONY_VERSION=2.7.*
   - SYMFONY_VERSION=2.8.*
+  - SYMFONY_VERSION=3.2.*
```

Comment:
What about Symfony 3.1? Is there any reason this bundle won't work with 3.1? If it does work, please add 3.1 here too.